cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/extensions/CimCharacteristicsMergingView.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.cgmes.conversion.extensions;
+
+import com.google.auto.service.AutoService;
+import com.powsybl.cgmes.conversion.elements.CgmesTopologyKind;
+import com.powsybl.commons.PowsyblException;
+import com.powsybl.iidm.mergingview.extensions.ExtensionMergingView;
+import com.powsybl.iidm.network.Network;
+
+/**
+ * @author Miora Ralambotiana <miora.ralambotiana at rte-france.com>
+ */
+@AutoService(ExtensionMergingView.class)
+public class CimCharacteristicsMergingView implements ExtensionMergingView<Network, CimCharacteristics> {
+
+    @Override
+    public Class<? extends CimCharacteristics> getExtensionClass() {
+        return CimCharacteristics.class;
+    }
+
+    @Override
+    public String getName() {
+        return "cimCharacteristics";
+    }
+
+    @Override
+    public void merge(Network extendable, CimCharacteristics extension) {
+        CimCharacteristics original = extendable.getExtension(CimCharacteristics.class);
+        if (original == null) {
+            extendable.newExtension(CimCharacteristicsAdder.class)
+                    .setCimVersion(extension.getCimVersion())
+                    .setTopologyKind(extension.getTopologyKind())
+                    .add();
+        } else {
+            checkCimVersions(original.getCimVersion(), extension.getCimVersion());
+            extendable.newExtension(CimCharacteristicsAdder.class)
+                    .setCimVersion(extension.getCimVersion())
+                    .setTopologyKind(getMergedTopologyKind(original.getTopologyKind(), extension.getTopologyKind()))
+                    .add();
```

Comment:
Why don't you update the topology kind of the existing extension, instead of creating a new one?